templates/etc/postfix/sender_canonical.j2

```diff
@@ -0,0 +1,9 @@
+# List of canonical mappings for envelope and header sender addresses
+# {{ ansible_managed }}
+
+{% for map in postfix_aws_sender_canonical_maps %}
+{% if map.comment is defined %}
+# {{ map.comment }}
+{% endif %}
```

Comment:
Will this generate an empty line between the comment and mapping? Probably it would be better to put whole if in one line and make new lines where necessary (i.e. after each comment-mapping block)